{
  "comments": [
    {
      "key": {
        "uuid": "895db3eb_a6092952",
        "filename": "plugins/tripleo-overcloud/plugin.spec",
        "patchSetId": 2
      },
      "lineNbr": 273,
      "author": {
        "id": 1008672
      },
      "writtenOn": "2017-11-28T06:08:16Z",
      "side": 1,
      "message": "Please, change to \u0027-\u0027 to be aligned with other variables.",
      "range": {
        "startLine": 273,
        "startChar": 26,
        "endLine": 273,
        "endChar": 27
      },
      "revId": "248f34ac06250e0ec6edbc4e300605c4bf1d7df5",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "73bb67ff_1ab82aca",
        "filename": "plugins/tripleo-overcloud/plugin.spec",
        "patchSetId": 2
      },
      "lineNbr": 273,
      "author": {
        "id": 1005532
      },
      "writtenOn": "2017-11-28T06:19:14Z",
      "side": 1,
      "message": "The problem I have is with the line in create_external_network.yml. I tried to use install.external.vlan but then it gets complicated because install.external dictionary doesn\u0027t exist. Do you have an idea ?",
      "parentUuid": "895db3eb_a6092952",
      "range": {
        "startLine": 273,
        "startChar": 26,
        "endLine": 273,
        "endChar": 27
      },
      "revId": "248f34ac06250e0ec6edbc4e300605c4bf1d7df5",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    }
  ]
}